### PR TITLE
add kickoff report blog

### PR DESCRIPTION
**Pull request by jcolomb**

### CODE REVIEW

File: content/blog/2021-10-07-kickoff/index.en.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+---
+
+On a Thursday afternoon (October 7th), the open.make project had his kickoff
+meeting on the Charité campus in Mitte. All participating research groups, a BUA
+representative and external partners presented their interests. We think it was
+a successful event, bringing us a broad overview of open hardware in academia
+and beyond. Originally planned as fully in-person, the kickoff meeting had to be
+shifted to a hybrid meeting, as two participants could not come.
+
+## Open.make partners
```
Comment (MIE5R0) [inline content/blog/2021-10-07-kickoff/index.en.md:24]:
```suggestion
## Open.make partners
```
```suggestion
## Open.make partners and attendees
```

**Review by MIE5R0 — APPROVED** [review]

File: content/blog/2021-10-07-kickoff/index.en.md (hunk 1 above)
Comment (jcolomb) [reply content/blog/2021-10-07-kickoff/index.en.md]: ok

File: content/blog/2021-10-07-kickoff/index.en.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+---
+
+On a Thursday afternoon (October 7th), the open.make project had his kickoff
+meeting on the Charité campus in Mitte. All participating research groups, a BUA
+representative and external partners presented their interests. We think it was
+a successful event, bringing us a broad overview of open hardware in academia
+and beyond. Originally planned as fully in-person, the kickoff meeting had to be
+shifted to a hybrid meeting, as two participants could not come.
+
+## Open.make partners and attendees
```
Comment (jcolomb) [inline content/blog/2021-10-07-kickoff/index.en.md:24]:
```suggestion
## Open.make partners 
```

doch keine gute Idee

**Comment by MIE5R0** [comment]
We will use HackMD to edit the document: https://hackmd.io/@UYy0pwpMSYO7rsopF2NuuQ/Sktamu0IY/edit

**Comment by MIE5R0** [comment]
The document is now back in dev and HackMD link is no longer accessible.

**Pull request merged**